lib/sneakers/handlers/maxretry.rb

```diff
@@ -82,6 +82,15 @@ def initialize(channel, queue, opts)
 
       end
 
+      def self.configure_queue name, opts
```

Comment:
I don't like asking for this but can we please use the existing code style for method definitions?